jbpm-wb-executor-service/jbpm-wb-executor-service-client/src/main/java/org/jbpm/workbench/es/client/editors/requestlist/RequestListPresenter.java

```diff
@@ -68,6 +67,7 @@
 import static org.dashbuilder.dataset.filter.FilterFactory.*;
 import static org.jbpm.workbench.common.client.PerspectiveIds.*;
 import static org.jbpm.workbench.es.model.RequestDataSetConstants.*;
+import static org.jbpm.workbench.pr.model.ProcessInstanceDataSetConstants.COLUMN_PROCESS_INSTANCE_ID;
```

Comment:
Is this import correct? Isn't this column already defined on `RequestDataSetConstants`?